lib/rules/function-calc-no-unspaced-operator/__tests__/index.js

```diff
@@ -153,6 +171,36 @@ testRule({
 			code: 'margin-top: calc(var(--some-variable)\r\n\t+ var(--some-other-variable));',
 			description: 'CRLF newline and tab before operator',
 		},
+		{
+			code: 'a { padding: 10px calc(calc(1px + 2px)* 3px); }',
+		},
+		{
+			code: 'a { padding: 10px calc(calc(1px* 2px) + 3px); }',
+		},
+		{
+			code: 'a { padding: 10px calc(1px /2); }',
+		},
+		{
+			code: 'a { padding: 10px calc(1px/ 2); }',
+		},
+		{
+			code: 'a { padding: 10px calc(1px *2); }',
+		},
+		{
+			code: 'a { padding: 10px calc(1px* 2); }',
+		},
+		{
+			code: 'a { top: calc(calc(1px + 2px)* 3px); }',
+		},
+		{
+			code: 'a { top: calc(calc(1px* 2px) + 3px); }',
+		},
+		{
+			code: 'a { top: calc(10px*var(--foo)); }',
+		},
+		{
+			code: 'a { top: calc(10px/var(--foo)); }',
+		},
```

Comment:
[note] The case raised a memory leak. See https://github.com/stylelint/stylelint/issues/6044#issuecomment-1112805765